test/metabase/analytics/snowplow_test.clj

```diff
@@ -129,7 +128,6 @@
 
 (deftest instance-creation-test
   (let [original-value (db/select-one-field :value Setting :key "instance-creation")]
-    (def my-original-value original-value)
```

Comment:
This got left in by accident in the last PR, oops

Reply:
Borkdude's changes to let us use clj-kondo should prevent this in the future.

Reply:
that'll be great